ChangeLog.d/mbedtls_ecp_export.txt

```diff
@@ -0,0 +1,3 @@
+Features
+   * Add a function to export ECP keypair parameters.
```

Comment:
Missed it the first time: please give the name of the function. Interested users shouldn't have to guess at it.

Reply:
Fixed as suggested.